sql/src/test/java/org/apache/druid/sql/calcite/NotYetSupported.java

```diff
@@ -88,7 +88,8 @@ enum Modes
     INCORRECT_SYNTAX(DruidException.class, "Incorrect syntax near the keyword"),
     // at least c7 is represented oddly in the parquet file
     T_ALLTYPES_ISSUES(AssertionError.class, "(t_alltype|allTypsUniq|fewRowsAllData).parquet.*Verifier.verify"),
-    RESULT_MISMATCH(AssertionError.class, "assertResultsEquals");
+    RESULT_MISMATCH(AssertionError.class, "assertResultsEquals"),
+    UNSUPPORTED_NULL_ORDERING(DruidException.class, "(A|DE)SCENDING ordering with NULLS (LAST|FIRST)");
```

Comment:
Stream of consciousness: Looking at these definitions and thinking about the comment above about the `DruidExceptionMatcher`.  I wonder if this enum shouldn't take a `Matcher<? extends Throwable>` instead of the class and message regex.

Reply:
Could be extended - but I don't really see the reason to do that right now: as so far the exception message matching was enough - it at some point it will need to optionally access some other things ; this can be extended/changed/etc

note that I don't want to give better access to these things from the annotation `@NotYetSupported` either ; so that these enum entries are more-or-less mandatory - so that they could also act as natural magnets for similar issues :)